e2e/staticpvc.go

```diff
@@ -189,6 +189,109 @@ func validateRBDStaticPV(f *framework.Framework, appPath string, isBlock, checkI
 	return err
 }
 
+func validateRBDStaticMigrationPV(f *framework.Framework, appPath string, isBlock, checkImgFeat bool) error {
```

Comment:
@humblec can we also consider improving this to take `mounter-type` parameter and act accordingly?
if so, it will be fairly simple to call this with krbd and rbd-nbd mounters from e2e/rbd.go.

Thanks!

Reply:
@pkalever yeah, can be done, will keep it for todo. thanks :+1:

Reply:
[Just repeating here for context]
If the tests are not part of the same #PR, then probably it means we cannot validate `get clusterID` for the NBD case.
if that is fine with others, then we can create an issue (or add to some tracker) for persistence (set bot rule to keep-alive) and move on.